packages/aws-cdk-lib/aws-ecs-patterns/lib/base/network-load-balanced-service-base.ts

```diff
@@ -132,10 +132,19 @@ export interface NetworkLoadBalancedServiceBaseProps {
   /**
    * Listener port of the network load balancer that will serve traffic to the service.
    *
-   * @default 80
+   * @default 80 or 443 with listenerCertificate provided
    */
   readonly listenerPort?: number;
 
+  /**
+   * Listener certificate list of ACM cert ARNs.
+   * If you provide a certificate, the listener's protocol will be TLS.
+   * If not, the listener's protocol will be TCP.
+   *
+   * @default - none
+   */
+  readonly listenerCertificate?: IListenerCertificate;
```

Comment:
should it be an array of IListenerCertificate to match the type of [the listener certificates property](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-elasticloadbalancingv2-listener.html#cfn-elasticloadbalancingv2-listener-certificates)